core/lib/types/src/commitment.rs

```diff
@@ -382,7 +382,7 @@ impl L1BatchAuxiliaryOutput {
             repeated_writes_compressed,
             system_logs_compressed,
             state_diffs_packed,
-        ) = if is_pre_boojum {
+        ) = if protocol_version.is_pre_boojum() {
```

Comment:
if its only used for the `l1_commit_data` then it can be removed but we just need to make sure it's not still being used to calculate gas limits, fees, or within metadata calculator under the hood

Reply:
i'd prefer to remove it during this upgrade and keep only pre boojum stage here and for everything after boojum use the input from vm